### PR TITLE
chore!: bump to v5

### PR DESCRIPTION
**Pull request by joshuaellis**

### CODE REVIEW

File: .changeset/config.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "https://unpkg.com/@changesets/config@2.3.1/schema.json",
   "commit": false,
-  "changelog": false,
+  "changelog": false,
```
Comment (innerdvations) [inline .changeset/config.json:4]:
```suggestion
  "changelog": false,
```
We might be missing a code formatter in the repo to enforce this

File: .changeset/little-drinks-eat.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@strapi/pack-up': major
+---
+
+line in the sand
```
Comment (innerdvations) [inline .changeset/little-drinks-eat.md:5]: what is this file? 😆

Comment (joshuaellis) [reply .changeset/little-drinks-eat.md:5]: they're changesets! https://github.com/changesets/changesets

File: .changeset/config.json (hunk 1 above)
Comment (joshuaellis) [reply .changeset/config.json]: yep, it wasn't targeting the file, fixed.

File: .changeset/little-drinks-eat.md (hunk 2 above)
Comment (innerdvations) [reply .changeset/little-drinks-eat.md:5]: Oh, nice, looks good!

**Review by innerdvations — APPROVED** [review]
LGTM

**Pull request merged**